duckduckgo/duckduckgo-privacy-extension
# Fix Amazon icon not showing up

**Reviewer:** @andrey-p 



## Description:
Amazon now shows up as Amazon.com, which breaks the icon.
I made a new shared template to be used wherever we need to normalize company names in order to display the icons. 
Removing the tld using a regex, besides just lowercasing the name, fixed the issue.
<img width="291" alt="screen shot 2018-04-10 at 1 49 55 pm" src="https://user-images.githubusercontent.com/3652195/38577634-8124c332-3ccf-11e8-86be-a4f02f323211.png">
<img width="305" alt="screen shot 2018-04-10 at 1 49 45 pm" src="https://user-images.githubusercontent.com/3652195/38577635-8143b706-3ccf-11e8-8326-c6cf775ad873.png">


## Steps to test this PR:
1. (Optional) force amazon to show up in the top blocked networks
2. Build and reload extension
3. In case 1. was skipped, make sure to load 30 pages. Some with amazon trackers are huffingtonpost.com, washingtonpost.com
4. Make sure the amazon icon shows up in the top blocked pills (if amazon is in the top 3)
5. Also check the tracker networks subview. The icon should show up there as well
6. Everything else should be unaffected


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
